README.md

```diff
@@ -273,6 +274,49 @@ your browser.
     'Client secret'.
 
 
+### 3.2. Facebook OAuth
+
+ 1. Go to https://developers.facebook.com/ and create an app.
+
+ 2. Add a Product 'Facebook Login' and select platform 'Web'. Enter 'Site URL' `https://{domain}/`.
+
+ 3. Go to 'Facebook Login' / 'Settings' and add 'Valid OAuth Redirect URIs':
+
+    	https://{domain}/accounts/facebook/login/callback/
+    	https://{domain}/sk/accounts/facebook/login/callback/
+    	https://{domain}/en/accounts/facebook/login/callback/
+
+    Where `{domain}` is your production domain or `127.0.0.1:8000` if in local development mode.
+    'Site URL' in form `127.0.0.1:8000` is invalid use `localhost:8000` instead.
+
+To use Facebook Login, you need to use https. Setup default protocol in `settings/common.py`.
+
+	ACCOUNT_DEFAULT_HTTP_PROTOCOL = u'https'
+
+To run a server with https you can use
+[RunServerPlus](https://django-extensions.readthedocs.org/en/latest/runserver_plus.html) from
+[django-extensions](https://github.com/django-extensions/django-extensions). To install run:
+
+	$ pip install django-extensions==1.0.3
```

Comment:
@martinmacko47 Nedaval som to ako dependecies, pretoze ich ku projektu netreba.

Reply:
@viliambalaz Ok. Mozno niekedy neskor mozeme pridat do `setup.py` moznost pri localhoste zapnut https. Zatial kludne takto.

Reply:
@viliambalaz Na druhe zamyslenie, mozno by bolo fajn radsej spravit sekciu "1.5. Viewing the site using https" a tam presunut popis ako rozbehat web na localhoste s https. Tu pri FB by sme uz iba spomenuli, ze web treba na localhoste spustit s https a linku na prislusnu kapitolu. Lebo zrejme aj pri inych veciach bude treba https. Tak nech to mame spisane na jednom mieste.

Reply:
:ok: